academy/golang-intro/1_install.md

```diff
@@ -0,0 +1,94 @@
+---
+title: "Go Introduction"
+order: 
+description: Get started with Golang
+tag: deep-dive
+---
+
+# Go Introduction
+
+Golang (Go) is an open-source, statically typed, and compiled programming language introduced by Google and first released in 2012. It has a BSD-style license. Its Git repository is located at [https://go.googlesource.com/go](https://go.googlesource.com/go).
+
+Start with the basics. In this section, you will install Go and learn some general things about it.
+
+
+## Install
+
+Installing Go depends on your machine. See the [Getting Started Page](https://golang.org/doc/install) and follow the instructions to install Go.
+
+<HighlightBox type="tip">
+
+[Gimme](https://github.com/travis-ci/gimme) is a useful script for Go.
+
+</HighlightBox>
+
+## Workflow
+
+Go has a different workflow from most other programming tools. You work with a single workspace and have all Go code contained in this workspace. The `GOPATH` environment variable specifies the location of your workspace. You can print it by typing `$ go env GOPATH` in your console. For more information about `GOPATH` type `$ go help gopath`.
```

Comment:
This make it seem like all your project needs to be in `GOPATH/src/` which is not the case.
You can have your code anywhere, but go will the dependencies in your `GOPATH`.